feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -171,7 +171,7 @@ func BuildBenchmarkingConfig(t *testing.T) *oc.Root {
 	lspBit.SetBit = ygot.Bool(false)
 	isisTimers := globalISIS.GetOrCreateTimers()
 	isisTimers.LspLifetimeInterval = ygot.Uint16(600)
-	isisTimers.LspRefreshInterval = ygot.Uint16(200)
+	isisTimers.LspRefreshInterval = ygot.Uint16(250)
```

Comment:
Only this will not work. Please also remove the deviation for gnmi 1.2

Reply:
The deviation itself can be removed since I added it only for Juniper

Reply:
I have removed deviation.

Reply:
It looks like this deviaiton is only being used here and now being removed.

If yes, please remove all references to this deviation. Ie. from all the following locations:
 - metadata.proto: https://github.com/openconfig/featureprofiles/blob/main/proto/metadata.proto#L277
 - deviations.go: https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/deviations.go#L561
 - metadata.textproto for all the tests where this deviation was used

Reply:
I did remove it in metadata.proto, deviations.go. 
AFAIK this is not used in metadata.textproto